packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1153,7 +1078,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
             switch (message.type) {
                 case ContainerMessageType.Attach:
-                    this.processAttachMessage(message, local, localMessageMetadata);
+                    this.datastores.processAttachMessage(message, local);
```

Comment:
Looking at the name, it gives an impression that we are calling processAttachMessage on multiple data stores. The call to createDetachedDataStoreCore also gives a similar feeling.
Maybe we should call this something else, like dataStoreManager?

Reply:
i think eventually datastores will be like a DDS. It should allow us to have multiple layers of datastores, so i'd rather not call it manager